fog/api/src/conversions.rs

```diff
@@ -14,3 +16,31 @@ impl From<&common::BlockRange> for fog_common::BlockRange {
         proto_block_range
     }
 }
+
+impl From<&ingest_common::IngestSummary> for mc_fog_types::ingest_common::IngestSummary {
+    fn from(
+        proto_ingest_summary: &ingest_common::IngestSummary,
+    ) -> mc_fog_types::ingest_common::IngestSummary {
+        let ingest_controller_mode = match proto_ingest_summary.mode {
+            ingest_common::IngestControllerMode::Idle => {
+                mc_fog_types::ingest_common::IngestControllerMode::IDLE
+            }
+            ingest_common::IngestControllerMode::Active => {
+                mc_fog_types::ingest_common::IngestControllerMode::ACTIVE
+            }
+        };
+        let ingress_pubkey: CompressedRistrettoPublic =
+            CompressedRistrettoPublic::try_from(proto_ingest_summary.get_ingress_pubkey()).unwrap();
```

Comment:
This should not call `unwrap()`. If it can fail, this should be a `try_from`.

Reply:
Done.